common/djangoapps/track/views/tests/test_segmentio.py

```diff
@@ -404,3 +405,207 @@ def test_video_event(self, name, event_type):
 
         self.assertEqualUnicode(actual_event, expected_event_without_payload)
         self.assertEqualUnicode(payload, expected_payload)
+
+    @data(
+        (1, "onSlideSeek"),
+        (-2, "onSlideSeek"),
+        (30, "onSkipSeek"),
+        (-30, "onSkipSeek")
```

Comment:
This doesn't test the renaming of `edx.video.seeked` to `edx.video.position.changed`.  This change was needed for android as well.
Suggestion: Why not combine both of these test methods into one?  You can pass the application name and version as ddt parameters.  The end comparison of the expected values are the same anyway, right?